chrome/content/coniks/context.js

```diff
@@ -0,0 +1,41 @@
+this.EXPORTED_SYMBOLS = ["AccountContext", "ConiksContext"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("chrome://otr/content/helpers.js");
+
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
+
+    if (opts.account === null) {
+      return null;
```

Comment:
This return scares me.  Are you doing any null checks anywhere?  Maybe it'd be better to throw or an explicit .validate()